t/APISIX.pm

```diff
@@ -509,6 +509,7 @@ _EOC_
     lua_shared_dict ext-plugin 1m;
     lua_shared_dict kubernetes 1m;
     lua_shared_dict tars 1m;
+    lua_shared_dict xds-route-config 1m;
```

Comment:
Should we sync the setting in `apisix/cli/ngx_tpl.lua` ?

Change `1m` to `10m`

Reply:
It is enough in the test.